.github/workflows/test_pypy.yml

```diff
@@ -0,0 +1,50 @@
+name: PyPy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [pypy2, pypy3]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      TOXENV: py
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }} on ${{ matrix.os }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: python -m pip install -U tox
+      - name: Install zic (Windows)
+        run: |
+          curl https://get.enterprisedb.com/postgresql/postgresql-9.5.21-2-windows-x64-binaries.zip --output $env:GITHUB_WORKSPACE\postgresql9.5.21.zip
+          unzip -oq $env:GITHUB_WORKSPACE\postgresql9.5.21.zip -d .postgresql
+        if: runner.os == 'Windows'
+      - name: Run updatezinfo.py (Windows)
+        run: |
+          $env:Path += ";$env:GITHUB_WORKSPACE\.postgresql\pgsql\bin"
+          ci_tools/retry.bat python updatezinfo.py
+        if: runner.os == 'Windows'
+      - name: Run updatezinfo.py (Unix)
+        run: ./ci_tools/retry.sh python updatezinfo.py
+        if: runner.os != 'Windows'
+      - name: Run tox
+        run: python -m tox
+      - name: Generate coverage.xml
+        run: python -m tox -e coverage
+      - name: Report coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: ./.tox/coverage.xml
+          name: ${{ matrix.os }}:${{ matrix.python-version }}
+          fail_ci_if_error: true
```

Comment:
Since Codecov has a server-side issue right now we could decide not to fail here, at least for now.